src/Suite.ts

```diff
@@ -15,14 +15,19 @@ async function writeTests() {
   const page = await TestBrowser.createPage(testBrowser, options);
   const stories = await TestBrowser.getStories(page);
   const storiesByComponent = groupBy(stories, 'componentTitle');
+  const patternRegex = new RegExp(options.pattern);
 
   describe(`[${options.browser}] accessibility`, function () {
     after(async function () {
       await TestBrowser.close(testBrowser);
     });
 
     each(storiesByComponent, (stories, componentTitle) => {
-      describe(componentTitle, () => {
+      const nameMatches = patternRegex.test(componentTitle);
+      const describeFn = nameMatches ? describe : describe.skip;
+      const testName = nameMatches ? componentTitle : `[skipped] ${componentTitle}`;
```

Comment:
adding this because otherwise it's not super clear that the whole file was skipped:
<img width="604" alt="Screen Shot 2020-10-26 at 12 35 50 PM" src="https://user-images.githubusercontent.com/15840841/97221480-2c3c3100-178a-11eb-8077-6190fb9798c0.png">

now instead it'll say "[skipped] input"

Reply:
Nice 👍